doc/design/abs_backup.md

```diff
@@ -0,0 +1,73 @@
+# Backups using Azure Blob Service (ABS)
+
+In a very similar fashion to backups using [S3](./s3_backup.md), we want to have an option to store backups in ABS.
+
+## Cluster configured with ABS backup
+
+To create a cluster with backups to ABS, we set `backup.storageType` to `"ABS"`.  Additionally, the ABS container for holding backups and the Kubernetes secret storing the Azure Storage account credentials must be supplied.  Both of these resources must exist prior to cluster creation (as will be discussed further below).
```

Comment:
"To create a cluster with backups to ABS,  set `backup.storageType` to `ABS`, supply the name of ABS container to `abs.absContainer`, and provide the  Kubernetes secret storing the Azure Storage account credentials to `abs. absSecret`. The latter two resources must exist prior to cluster creation."